src/main/cluster.ts

```diff
@@ -88,12 +88,7 @@ export interface ClusterState {
 export class Cluster implements ClusterModel, ClusterState {
   /** Unique id for a cluster */
   public readonly id: ClusterId;
-  /**
-   * Kubectl
-   *
-   * @internal
-   */
-  public kubeCtl: Kubectl;
+  #kubeCtl: Kubectl;
```

Comment:
We haven't really used `#`, maybe change to `private`?

Reply:
sure